libyul/AsmParser.cpp

```diff
@@ -78,14 +108,62 @@ unique_ptr<Block> Parser::parse(std::shared_ptr<Scanner> const& _scanner, bool _
 	return nullptr;
 }
 
+langutil::Token Parser::advance()
+{
+	auto const token = ParserBase::advance();
+	if (m_useSourceLocationFrom == UseSourceLocationFrom::Comments)
+		fetchSourceLocationFromComment();
+	return token;
+}
+
+void Parser::fetchSourceLocationFromComment()
+{
+	solAssert(m_charStreamMap.has_value(), "");
+
+	if (m_scanner->currentCommentLiteral().empty())
+		return;
+
+	static std::regex const lineRE = std::regex(
+		R"~~~((^|\s+)@src\s+(\d+):(-1|\d+):(-1|\d+)\s*$)~~~",
```

Comment:
I think it's fine to use
```suggestion
		R"~~~((^|\s+)@src\s+(\d+):(-1|\d+):(-1|\d+)(\s+|$))~~~",
```

Reply:
This is caussing all tests to fail ;-(